zerver/openapi/zulip.yaml

```diff
@@ -3095,6 +3095,458 @@ paths:
+        schema:
+          type: integer
+          nullable: true
+        example: 1
+        required: false
+      - name: invite_to_stream_policy
+        in: query
+        description: |
+          An integer that defines who can add users to stream.
+          * 1: Admins and memebers can add users to streams.
+          * 2: Only admins can add users to streams.
+          * 3: Admins and full members can add users to streams.
+        schema:
+          type: integer
+          nullable: true
+        example: 1
+        required: false
+      - name: user_group_edit_policy
+        in: query
+        description: |
+          An integer that defines who can create and manage user groups.
+          * 1: Admins and memebers can create and manage user groups.
+          * 2: Only admins can create and manage user groups.
+        schema:
+          type: integer
+          nullable: true
+        example: 2
+        required: false
+      - name: private_message_policy
+        in: query
+        description: |
+          An integer that defines who can use private messages. This feature is
+          beta and it is only available to organization owners and admins.
+          * 1: Admins, members, and guests are allowed to use private messages.
+          * 2: Private messages disabled.
+        schema:
+          type: integer
+          nullable: true
+        example: 1
+        required: false
+      - name: email_address_visibility
+        in: query
+        description: |
+          An integer that defines who can access user email addresses.
+          * 1: Admins, members, and guests.
+          * 3: Admins only.
+          * 4: Nobody.
+        schema:
+          type: integer
+          nullable: true
+        example: 1
+        required: false
+      - name: default_twenty_four_hour_time
+        in: query
+        description: |
+          A boolean that indicates whether the twenty-four-hour-time or the
+          twelve-hour-time.
+        schema:
+          type: boolean
+          nullable: true
+        example: true
+        required: false
+      - name: video_chat_provider
+        in: query
+        description: |
+          An integer that indicates which is the organization video chat provider.
+          * 0: None.
+          * 1: Jitsi meet.
+          * 3: Zoom.
+        schema:
+          type: integer
+          nullable: true
+        example: 1
+        required: false
+      - name: default_code_block_language
+        in: query
+        description: |
+          A string that indicates the default language for code blocks.
+        content:
+          application/json:
+            schema:
+              type: string
+              nullable: true
+            example: python
+        required: false
+      - name: digest_weekday
+        in: query
+        description: |
+          An integer that indicates the day that the weekly digest emails will
+          be sent to inactive users.
```

Comment:
The bulleted lists are all coming in one line in the browser. You can have a look at `update-stream` for the bulleted lists in parameters.